tests/testdata/default_config.yaml

```diff
@@ -19,6 +19,9 @@ chat:
   # Filepath of a dialog session file.
   # Default: None
   session:
+  # Custom system prompt. If not provided, it will use the default context.
+  # Default: ''
+  system_prompt: ''
```

Comment:
Do we really want the default system system prompt to be set to an empty string? It seems like a better default would be simply not including it / having it set to `None`.

Reply:
The default in pydantic is `None` (https://github.com/instructlab/instructlab/blob/main/src/instructlab/configuration.py#L527). Not sure why it's rendering to an empty string here.

Reply:
thanks, updated